releases/goerli/1.9.6/contracts.json

```diff
@@ -0,0 +1,261 @@
+  "PIP_RWA001": "0x95282c2cDE88b93F784E2485f885580275551387",
+  "MCD_JOIN_RWA001_A": "0x088D6b3f68Bc4F93F90006A1356A21145EDD96E2",
+  "RWA001_A_URN": "0xF1AAB03fc1d3588B5910a960f476DbE88D304b9B",
+  "RWA001_A_INPUT_CONDUIT": "0x4145774D007C88392118f32E2c31686faCc9486E",
+  "RWA001_A_OUTPUT_CONDUIT": "0x969b3701A17391f2906d8c5E5D816aBcD9D0f199",
+  "RWA002": "0x09fE0aE289553010D6EcBdFF98cc9C08030dE3b8",
+  "PIP_RWA002": "0xF1E8E72AE116193A9fA551beC1cda965147b31DA",
+  "MCD_JOIN_RWA002_A": "0xc0aeE42b5E77e931BAfd98EAdd321e704fD7CA1f",
+  "RWA002_A_URN": "0xD6953949b2B4Ab5Be19ed6283F4ca0AaEDDffec5",
+  "RWA002_A_INPUT_CONDUIT": "0x1d3402B809095c3320296f3A77c4be20C3b74d47",
+  "RWA002_A_OUTPUT_CONDUIT": "0x1d3402B809095c3320296f3A77c4be20C3b74d47",
+  "RWA003": "0x5cf15Cc2710aFc0EaBBD7e045f84F9556B204331",
+  "PIP_RWA003": "0x27E599C9D69e02477f5ffF4c8E4E42B97777eE52",
+  "MCD_JOIN_RWA003_A": "0x83fA1F7c423112aBC6B340e32564460eDcf6AD74",
+  "RWA003_A_URN": "0x438262Eb709d47b0B3d2524E75E63DBa9571962B",
+  "RWA003_A_INPUT_CONDUIT": "0x608050Cb6948A9835442E24a5B1964F76fd4acE4",
+  "RWA003_A_OUTPUT_CONDUIT": "0x608050Cb6948A9835442E24a5B1964F76fd4acE4",
+  "RWA004": "0xA7fbA77c4d18e12d1F385E2dcFfb377c9dBD91d2",
+  "PIP_RWA004": "0x3C191d5a74800A99D8747fdffAea42F60f7d3Bff",
+  "MCD_JOIN_RWA004_A": "0xA74036937413B799b2f620a3b6Ea61ad08F1D354",
+  "RWA004_A_URN": "0x1527A3B844ca194783BDeab8DF4F9264D1A9F529",
+  "RWA004_A_INPUT_CONDUIT": "0x551837D1C1638944A97a6476ffCD1bE4E1391Fc9",
+  "RWA004_A_OUTPUT_CONDUIT": "0x551837D1C1638944A97a6476ffCD1bE4E1391Fc9",
+  "RWA005": "0x650d168fC94B79Bb16898CAae773B0Ce1097Cc3F",
+  "PIP_RWA005": "0xa6A7f2408949cAbD13f254F8e77ad5C9896725aB",
+  "MCD_JOIN_RWA005_A": "0xc5052A70e00983ffa6894679f1d9c0cDAFe28416",
+  "RWA005_A_URN": "0x047E68a3c1F22f9BB3fB063b311dC76c6E308404",
+  "RWA005_A_INPUT_CONDUIT": "0x8347e6e08cAF1FB63428465b76BafD4Cf6fcA2e1",
+  "RWA005_A_OUTPUT_CONDUIT": "0x8347e6e08cAF1FB63428465b76BafD4Cf6fcA2e1",
+  "RWA006": "0xf754FD6611852eE94AC0614c51B8692cAE9fEe9F",
+  "PIP_RWA006": "0xA410A66313F943d022b79f2943C9A37CefdE2371",
+  "MCD_JOIN_RWA006_A": "0x5b4B7797FC41123578718AD4E3F04d1Bde9685DC",
+  "RWA006_A_URN": "0xd0d2Ef46b64C07b5Ce4f2634a82984C1B3804C22",
+  "RWA006_A_INPUT_CONDUIT": "0xd2Ef07535267D17d2314894f7821A43e9700A02e",
+  "RWA006_A_OUTPUT_CONDUIT": "0xd2Ef07535267D17d2314894f7821A43e9700A02e",
+  "PROXY_PAUSE_ACTIONS": "0x8D1187FCa9A104211bd25c689C08718AD8730C83",
+  "PROXY_DEPLOYER": "0xc9476Fd378de5b0de5C4280D4323f6F89f723c15",
+  "LERP_FAB": "0xbBD821c291c492c40Db2577D9b6E5B1bdAEBD207",
+  "GUNIV3DAIUSDC1": "0xc5D83e829Ecdce4d67645EE1a1317451e0b4c68d",
+  "PIP_GUNIV3DAIUSDC1": "0xF953cdebbbf63607EeBc556438d86F2e1d47C8aA",
+  "MCD_JOIN_GUNIV3DAIUSDC1_A": "0xFBF4e3bB9B86d24F91Da185E6F4C8D903Fb63C86",
+  "MCD_CLIP_GUNIV3DAIUSDC1_A": "0xFb98C5A49eDd0888e85f6d2CCc7695b5202A6B32",
+  "MCD_CLIP_CALC_GUNIV3DAIUSDC1_A": "0x4652E3a6b4850a0fE50E60B0ac72aBd74199D973"
```

Comment:
I'd put these ones right after the last UNIV2 one.

Reply:
done